.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+updates:
+  - package-ecosystem: "cargo"
+    directory: "/"
+    schedule:
+      interval: "daily"
+      time: "02:00" # UTC
+    labels:
+      - "domain: deps"
+    commit-message:
+      prefix: "* RoboYak deps"
```

Comment:
I guess I missed this: seems like it might be copypasta? It should probably be changed

Reply:
Nothing wrong with it really? just how I do it elsewhere... I like clear prefixed commit messages and put labels so I can organise my PRs appropriately with them.

Reply:
Aah, ok. I was confused by the name "RoboYak". I assumed it was some other project you had copied and pasted the Dependabot config from.

Perhaps we could change it to something a little less surprising like "Dependabot"?